src/PackageVersions/Installer.php

```diff
@@ -177,6 +177,12 @@ private static function getVersions(Locker $locker, RootPackageInterface $rootPa
             );
         }
 
-        yield $rootPackage->getName() => $rootPackage->getPrettyVersion() . '@' . $rootPackage->getSourceReference();
+        $rootVersion = $rootPackage->getPrettyVersion() . '@' . $rootPackage->getSourceReference();
+
+        foreach ($rootPackage->getReplaces() as $replace) {
+            yield $replace->getTarget() => $rootVersion;
```

Comment:
Seems wrong to me: the replaced package is not installed, so this is not its version.

Reply:
In most cases replace directive is used in conjunction with self.replace version. Actually I didn't yet seen a usage of this using it differently and it's discouraged to use it differently. I didn't want to hardcode there such magic string, but I will if you think it's better. Should I? Also not sure I can get a reference part for such packages yet, I'm on mobile.

Reply:
I'm fine with showing a version, but that string should somehow show that this is a replacement. Adding the replacement package name to the version would be sufficient, IMO.

Reply:
Ok I'll change it to add there literal value (so e.g. `self.replace), since this PR is just because of CI, where it won't matter. But `@` suffix I guess will need to stay, as not having this would break packages like https://github.com/Jean85/pretty-package-versions